core/node/storage/testdata/migrations/000006_fixed_stream_partitions.up.sql

```diff
@@ -0,0 +1,41 @@
+DO $$
```

Comment:
all previous schema files are outdated: they are all moved into one final schema in generation 5.